frontend/server/src/Controllers/Problem.php

```diff
@@ -1974,6 +2011,19 @@ private static function invalidateCache(
         );
     }
 
+    private static function invalidateProblemsAclCacheForIdentity(
+        \OmegaUp\DAO\VO\Identities $identity
+    ): void {
+        $identityId = $identity->identity_id;
+        $userId = $identity->user_id ?? null;
+        $cacheKey = "{$identityId}-" . ($userId ?? 'null');
```

Comment:
Si el `$userId` sólo lo vas a utilizar para el propósito de la clave del caché, quizás puedas simplificar un poco:
```suggestion
        $userId = $identity->user_id ?? 'null';
        $cacheKey = "{$identityId}-{$userId}";
```
De esta forma no haces dos veces el null coalescing operator, aunque la cadena puede estar vacía para no inventar cadenas mágicas

Reply:
Vale, hago el cambio.